### DISCUSSION TITLE
is having cockpit-pcp collectin metrics running all the time an issue?

### DISCUSSION THREAD

**Discussion by emaayan** [cockpit-project/cockpit#17180] [Q&A] [answered]
i'm wondering if running pcp all the time can create performance issues in production, any suggestion about that?

**Comment by martinpitt** [comment, marked as answer]
Well, that's its main purpose. If your machine can't afford some 1 or 2 percent measurement overhead, or you don't need metrics, then by all means disable them. But in many environments the small performance price is worth the ability to find and debug performance issues easily.

**Comment by emaayan** [comment]
holly crap, totally missed the discussion part, kept wondering how i can ask a question without using an issue tracker,
if my machine can't afford 1-2 percent, i'd need a bigger boat, this is part of an OVA that's meant to be distributed, and i wanted to have it as a part of a troubleshooting tool. 
the same way i wanted to use the report report-sos plugin (even though it produces a zip file i can't fully extract, and i can't configure it to look into other folders)
